pulp_ansible/app/urls.py

```diff
@@ -225,8 +241,8 @@
 v3_plugin_urls = [
     # path:var captures /, so it has to have something at the end to make it work
     # correctly.
-    path("content/<path:distro_base_path>/collections/", include(v3_collection_urls)),
-    path("content/<path:distro_base_path>/namespaces/", include(namespace_urls)),
+    path("content/<basepath:distro_base_path>/collections/", include(v3_collection_urls)),
+    path("content/<basepath:distro_base_path>/namespaces/", include(namespace_urls)),
```

Comment:
So this is an issue for collections named "namespaces" too. What about a namespace called "collections" or "namespaces"?